aos/aos_milestones/IMpg/check_impg_run.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>The units on the hexapod plots are wrong.</p><p>instead of mm, we should have micron and degree.</p><p>Consider plotting x,y,z for the two hexapods on a subpanel, and u,v for the two hexapods on another, perhaps?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>done</p>